Remove internal build of TBB. Use standard tbb instead of tbb_preview.
Original Author: micketeer@gmail.com

diff --git a/usher-0.5.0.orig/CMakeLists.txt b/usher-0.5.0/CMakeLists.txt
index 66a9032..4ce3949 100644
--- a/usher-0.5.0.orig/CMakeLists.txt
+++ b/usher-0.5.0/CMakeLists.txt
@@ -38,9 +38,7 @@ find_package(MPI REQUIRED)
 include_directories(SYSTEM ${MPI_INCLUDE_PATH})
 include_directories(${Protobuf_INCLUDE_DIRS})
 
-include(${TBB_DIR}/cmake/TBBBuild.cmake)
-tbb_build(TBB_ROOT ${TBB_DIR} CONFIG_DIR TBB_DIR MAKE_ARGS tbb_cpf=1)
-find_package(TBB REQUIRED tbbmalloc tbbmalloc_proxy tbb_preview)
+find_package(TBB REQUIRED tbbmalloc tbbmalloc_proxy tbb)
 
 FIND_PACKAGE(Boost COMPONENTS program_options iostreams filesystem date_time REQUIRED)
 
